cocotb/decorators.py

```diff
@@ -281,13 +283,19 @@ def abort(self, exc):
         """
         return self._force_outcome(outcomes.Error(exc))
 
+    def sort_name(self):
+        if self.stage is None:
+            return "%s.%s" % (self.module, self.funcname)
+        else:
```

Comment:
style: avoid else after return.

Reply:
I'd disagree with this in this particular instance, since the if and else clauses are pretty symmetric. Not a big deal either way.

Reply:
I have to agree with Eric. In this case it's a statement that could be written as a ternary, but would otherwise be too long. Having separate branches that end with the last statement being similar encodes the same meaning. However, I know many linters (e.g. pylint) disagree.